specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/MigrationValidation.json

```diff
@@ -0,0 +1,415 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Data Migration Service Resource Provider",
```

Comment:
Search for 'Azure Data Migration Service' and replace with 'Azure Database Migration Service

Reply:
Done.